webroot/js/cake-core.js

```diff
@@ -0,0 +1,22 @@
+var elements = document.getElementsByClassName('cake-core-postLink');
```

Comment:
@markstory Ok got it! :laughing: Yeah I added cake-core.js and cake-core.css in the default layout.

@ankr Yeah, it should wait for DOM ready. As it is not that trivial to implement it while supporting every browser without native javascript my suggestion was to add the script loading at the end of the html document. But I found some source code to support every browser:
http://javascript.info/tutorial/onload-ondomcontentloaded
I added this into the js file now. But unfortunately I can't test it for older browsers.